idd/Energy+.idd.in

```diff
@@ -108717,10 +108717,20 @@ Output:JSON,
 Output:SQLite,
        \memo Output from EnergyPlus can be written to an SQLite format file.
        \unique-object
-  A1 ; \field Option Type
+  A1 , \field Option Type
        \type choice
        \key Simple
        \key SimpleAndTabular
+  A2 ; \field Unit Conversion for Tabular Data
+       \type choice
+	   \note Unit conversion option used when writing SQLite Tabular Data
```

Comment:
Add to the note here on which tables have the unit conversion applied to them.

Reply:
Done.